app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class='shopping-list'>
+ <h1 class='public-recipes-title'>Public Recipes</h1>
+ <% @public_recipes.each do |public_recipe| %>
+   <% if can? :delete, public_recipe %>
+      <%= link_to "Delete", recipe_path(public_recipe), method: :delete, data: { confirm: "Are you sure?" } %>
+    <% end %>  
+<%= link_to recipe_path(public_recipe) do %>   
+<div class='recipe-container'>
+<section class='recipe-info'>
+   <h3 class='recipe-1'><%= public_recipe.name %></h3>
+   <p class='recipe-btn'>By:<%= public_recipe.user.name %></p>
+</section> 
+<section class='public-total'>   
+   <p class='recipe-description'>Total food Items: <%= public_recipe.total_quantity %></p>
+   <p class='recipe-description'>Total price: $<%= public_recipe.total_price %></p>
+</section> 
+</div>
+<% end %>
+<% end %>
+
```

Comment:
![image](https://github.com/ismayito/Rails-recipe-app/assets/59397159/0209d7e0-1108-42a6-8057-26fcbc470226)


![Uploading image.png…]()

---

- Your public recipe is currently empty, it should reflect the wireframe above, please, kindly implement it. I understand and admire the hard work you've put in, working on the project alone, however, you can do more as I've been in your shows too. please, see this as an opportunity to expand your learning. :muscle:

Reply:
**KANBAN BOARD**

- In your [kanban board](https://github.com/users/ismayito/projects/3/views/1?template_dialog_tab=featured&filterQuery=assignee%3Aismayito), please, kindly move your cards accordingly. You've finished all your tasks, so they should be marked Done. please, fix this and close all open issues pending the completion of this project.